clients/p4rt-ctl/p4rt-ctl.in

```diff
@@ -769,6 +791,54 @@ def p4ctl_show(client, bridge):
 def p4ctl_set_pipe(client, bridge, device_config, p4info):
     client.set_fwd_pipe_config(p4info, device_config)
 
+TUNSETIFF = 0x400454CA
```

Comment:
Can these be moved inside the class where the variables are used? Is there a reason for keeping them at a global level?

Reply:
Yes, done.